modules/util/source-formatter/src/main/java/com/liferay/source/formatter/BaseSourceProcessor.java

```diff
@@ -773,6 +773,20 @@ protected String getContent(String fileName, int level) throws IOException {
 		return StringPool.BLANK;
 	}
 
+	protected String getCopyright() throws Exception {
+		String copyright = getContent(
+			sourceFormatterArgs.getCopyrightFileName(), PORTAL_MAX_DIR_LEVEL);
+
+		if (Validator.isNotNull(copyright)) {
+			return copyright;
+		}
+
+		Class<?> clazz = getClass();
+
+		return StringUtil.read(
```

Comment:
@hhuijser 
I think you'll need to move the copyright.txt as well. This line is grabbing it relative to the class location.

Currently it's located here:

resources/com/liferay/source/formatter/checks/dependencies/copyright.txt